README.md

```diff
@@ -42,6 +42,31 @@ const sdk = await new AppExtensionsSDK({ identifier: '123abc' })
   .initialize({ size: { height: 500 } });
 ```
 
+### Without module bundler
+
+Initialization without a module bundler is possible by adding the following script to your HTML page from jsDelivr CDN:
+
+```HTML
+<head>
+  <script src="https://cdn.jsdelivr.net/npm/@pipedrive/app-extensions-sdk@0.8.0/dist/index.umd.js"></script>
```

Comment:
Since we follow semver, I think it's safe to install latest non-major version so we don't have to manually update this. I guess installing the sdk with package.json works same way

```suggestion
  <script src="https://cdn.jsdelivr.net/npm/@pipedrive/app-extensions-sdk@0/dist/index.umd.js"></script>
```